src/common/shutdown_state.c

```diff
@@ -202,14 +212,17 @@ shutdown_state_check(struct shutdown_state *curr_sds,
 		return 0;
 	}
 	if (dirty == 0) {
-		/* an ADR failure but the pool was closed */
 		CORE_LOG_WARNING(
-			"an ADR failure was detected but the pool was closed - SDS will be reinitialized");
+			is_uuid_correct ?
+			"an ADR failure was detected but the pool was closed - SDS will be reinitialized" :
+			"the pool has moved to a new location but it was closed properly - SDS will be reinitialized");
```

Comment:
Probably the final nit-pick. 😉 I would like to take the common part of both messages separate.

```c
CORE_LOG_WARNING("%s - SDS will be reinitialized", is_uuid_correct ?
	"an ADR failure was detected but the pool was closed" :
	"the pool has moved to a new location but it was closed properly");
```